.github/ISSUE_TEMPLATE.md

```diff
@@ -1,18 +1,20 @@
 ## Description
-Add a brief and meaningful description.
+<!-- Add a brief and meaningful description. -->
 
 ## Expected Behavior
+<!-- Describe the expected behaviour. -->
 
 ## Actual Behavior
+<!-- Describe the current/actual behaviour. -->
 
-## Steps to Reproduce
-1.
-2.
+## Environment
 
-## Specifications
-  - Browser:
-  - Browser version:
-  - OS: 
+* Operating system: (E.g RHEL 7.6 )
+* OpenShift version:
+<!-- Run the command `oc version` and add the result here. -->
+* Ansible version:
+<!-- Run the command `ansible --version` and add the result here. -->
+* Project Version/Tag: (E.g release-1.0.1)
```

Comment:
As we are not using semver, the version is still 1.x, but there have been a multitude of commits to master since then. It might be worthwhile to use something like the git hash or branch date.